docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+| normalized expression matrix | `adata.X`  | Numpy array or scipy sparse CSC matrix | Used for visualization in cellxgene explorer                   |
+| cell level metadata | `adata.obs` | Pandas dataframe | Categorical and continuous metadata shown in left and right cellxgene explorer sidebars respectively (can be used to color cells) |
+| variable/feature level metadata | `adata.var`   | Pandas dataframe|                                                             |
+| embedding                       | `adata.obsm`                           | Numpy array | Must start with the prefix 'X_' (i.e. adata.obsm['X_UMAP'])               |
+
+</div>
+
+<br/>
+
+<div align="center">
+  <b> Table: </b> Required data and `AnnData` object structure
+</div>
+
+<br/>
+
+<!---
+**Note:** Some assays, such as scATACseq or other epigenetic assays may not have a standardized way of representing a raw count matrix. While we still require there to be a matrix in location annotated as `raw`, some alternative options exist for these assays such as putting the un-normalized gene activity matrix in the `raw` slot. Since `adata.raw.X` can take matrices that are of a different dimensionality than `adata.X`, you could potentially put in a peak x cell matrix in this location instead. We leave this up to the author's discretion, but are happy to chat about options.
+--->
+
+**Note:** In addition to these data, other representations of the expression matrix (alternative normalizations, SCTransform, corrected counts from SCTransform or background corrected counts) can all be stored as `layers` in your `AnnData` object (as long as they maintain the same dimensionality of the main expression matrix used for visualization).
+
+**Note:** Information which pertains to the cellxgene schema will be stored in the `adata.uns` and `adata.obs` slots of the `AnnData` object and will be discussed in the [next section](#schema-definition).
+ 
+### Format conversion
+ 
+There are a handful of tools that can be used to convert different single cell formats (`seurat`, `loom`, `sce`, `cds`) to `AnnData`. We have had the most success with the R package [sceasy](https://github.com/cellgeni/sceasy). For relatively straightforward conversions, it is suitable to use the `sceasy::convertFormat()` function as specified in the sceasy documentation. For scenarios where you have multiple assays in the same object (as you may encounter in the `seurat` toolchain) it is recommended to check out the function definitions of the methods that are called by `convertFormat()`. For instance, if you look at the source code for `sceasy::seurat2anndata()`, you will see that the extra parameters `assay`, `main_layer`, and `transfer_layers` can be specified for control over which elements of the `seurat` object are carried into the `AnnData` object. Check out the following function definitions and potentially use as a starting point for more customized conversions:
```

Comment:
Who is "We" ?

Show some examples - notebooks?

Reply:
"We" is czi single cell curation team

I'm inclined to agree, but the most common conversions might be already fulfilled by examples in `sceasy` and more specific conversion would be out of scope for this tutorial